.gitignore

```diff
@@ -18,6 +18,7 @@ notebooks/01-use-cases/finance/risk-management/credit-risk/xva/data-prev/*.csv
 notebooks/01-use-cases/finance/risk-management/credit-risk/xva/data/*.csv
 notebooks/01-use-cases/finance/risk-management/credit-risk/xva/what-if/*.csv
 notebooks/01-use-cases/finance/risk-management/credit-risk/ifrs9/lending-club-data/*.csv
```

Comment:
@printhellohetal Was looking at our `gitignore` file to prevent processed data files from being checked in from the credit card analytics data processing notebook and saw these lines. Looks like they haven't been registering due to the deprecated file paths, therefore going to fix this, but also noting this as a symptom of an overarching issue that I've created a ticket for here: https://activeviam.atlassian.net/browse/AC-85